index.js

```diff
@@ -19,15 +19,21 @@ const staticify = (root, options) => {
             includeAll: opts.includeAll || false,
             shortHash: opts.shortHash || true,
             pathPrefix: opts.pathPrefix || '/',
+            maxAgeNonHashed: opts.maxAgeNonHashed || 0,
             sendOptions: opts.sendOptions || {}
         };
 
         defaultOptions = Object.assign(defaultOptions, opts);
 
+        defaultOptions.sendOptions.root = root;
```

Comment:
Can't we move these 2 into `setOptions`?

Reply:
I still feel like the following assignments aren't all needed.

We just have one options Object. Can't you put the new options there and just base your logic on `maxAge`?

Reply:
If there was only one options object, the logic in `serve` would have to change `maxAge` for non-hashed assets and after calling `send` reset `maxAge` back to original value which may be confusing and introduces unnecessary overhead with every call.

The reason for this pull request is to resolve issues with a single options object's `maxAge` being overwritten when a non-hashed asset is requested. Such call resets `maxAge` to `0` and all subsequent requests started to use `MAX_AGE`.